docs/getting-started/user-guides/paymetheus.md

```diff
@@ -0,0 +1,60 @@
+# **Paymetheus** #
+
+Paymetheus is a simple to use graphical wallet for Decred. With it, you can send and receive DCR, 
+purchase tickets for [PoS voting](/mining/proof-of-stake.md), get a history of all your transactions and more.
+About the only thing Paymetheus doesn't do is PoS voting.
+
+---
+
+## **Download and Install** ##
+The first thing you'll need to do is get a copy of Paymetheus. You will find the downloads [here](https://github.com/decred/decred-binaries/releases). 
+Download the file for your operating system and install. Once you're done, run the 'Decred' program.
+You'll notice a program called "Paymetheus Standalone". This does not include the 'dcrd' program 
+required to connect to the Decred network, so don't use this one.
+
+---
+
+## **Start Paymetheus** ##
+You're now ready to start using Decred! Start Paymetheus Standalone. You will now see the connection screen:  
+
+![Paymetheus connection screen](../../img/Paymetheus-dcrd-login.png)  
+
+Paymetheus comes with a dcrd server built in so just press Continue.
```

Comment:
the installer comes with dcrd, and the launcher (branded as Paymetheus in the start menu, but it is not paymetheus itself -- that is Paymetheus Standalone) will launch it.

Paymetheus is *just* a client, no dcrd at all, and even if we don't provide or give details instructions on how to use the standalone version without the launcher, it would be useful to at point out the architecture at least a little bit so that if people did want to run dcrd on a more appropriate computer than their desktop or laptop, they would know they could.